v2_test.go

```diff
@@ -77,11 +77,11 @@ func testV2[T VerifiedRequest[exampleAuthData]](t *testing.T, newV2 func(Credent
 		assert.NoError(t, err)
 		assert.Equal(t, body, b)
 	})
-	t.Run("List", func(t *testing.T) {
-		req := httptest.NewRequest(http.MethodGet, "https://awsexamplebucket1.us-west-1.s3.amazonaws.com/?prefix=photos&max-keys=50&marker=puppy", nil)
+	t.Run("List versions", func(t *testing.T) {
```

Comment:
do you mind copying this change as a separate test with the added "versions" to the query string? (this _List_ one intends to match the same one from AWS docs)